ms-ledger/res/configuration.toml

```diff
@@ -21,25 +21,8 @@ Host = "localhost"
 Port = 8500
 Type = 'consul'
 
-[MessageBus]
-Type = 'zero'
-    [MessageBus.PublishHost]
-        Host = '*'
-        Port = 5564
-        Protocol = 'tcp'
-    [MessageBus.SubscribeHost]
-        Host = 'localhost'
-        Port = 5563
-        Protocol = 'tcp'
-
-[Logging]
-EnableRemote = false
-File = ''
-
-[Binding]
+[Trigger]
 Type="http"
-SubscribeTopic=""
-PublishTopic=""
 
-[ApplicationSettings]
+[AppCustom]
```

Comment:
Since `http://localhost:48095/inventory` is a simple string, `[ApplicationSettings]` is more appropriate.

Using Custom config is overly complex for this simple setting.

Use Custom config when types are not `string` and/or the config could benefit from structure rather than one level or the values can be update dynamically (have to watch for changes)